Core/Src/main.c

```diff
@@ -114,7 +113,7 @@ int main(void)
   MX_GPIO_Init();
   MX_DMA_Init();
   MX_USART3_UART_Init();
-  MX_USB_DEVICE_Init();
+  MX_USB_DEVICE_Init(); // Might be a good idea to add the delay in here instead of in the loop()

```

Comment:
You can't add stuff outside of `USER CODE` blocks. It will be overwritten the next time code is generated by CubeMX. Not an issue, just an FYI.

Reply:
Removed in commit e1e9d70